packages/next/build/webpack-config.ts

```diff
@@ -52,6 +52,7 @@ export default function getBaseWebpackConfig (dir: string, {dev = false, isServe
       ...nodePathList // Support for NODE_PATH environment variable
     ],
     alias: {
+      ['next-app']: appPath,
```

Comment:
`private-next-app` to be in line with `PAGES_DIR_ALIAS` / `DOT_NEXT_ALIAS`

Reply:
You'll also want to remove _app from client compilation `entry`, otherwise it's doing needless compilation 🤔